server/etcdmain/config.go

```diff
@@ -291,6 +291,7 @@ func newConfig() *config {
 
 	// unsafe
 	fs.BoolVar(&cfg.ec.UnsafeNoFsync, "unsafe-no-fsync", false, "Disables fsync, unsafe, will cause data loss.")
+	fs.BoolVar(&cfg.ec.UnsafeAllowClusterVersionDowngrade, "unsafe-allow-cluster-version-downgrade", cfg.ec.UnsafeAllowClusterVersionDowngrade, "Enables only one single minor cluster version downgrade at a time automatically.")
```

Comment:
`"true" to allow cluster version downgrade, because newer minor versions may introduce incompatible feature changes.`